addons/binding/org.openhab.binding.solarlog/OSGI-INF/SolarLogHandlerFactory.xml

```diff
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright (c) 2010-2018 by the respective copyright holders. All rights reserved. This program and the accompanying 
-	materials are made available under the terms of the Eclipse Public License v1.0 which accompanies this distribution, and 
-	is available at http://www.eclipse.org/legal/epl-v10.html -->
+<!--
+
+	Copyright (c) 2010-2018 by the respective copyright holders.
+
+	All rights reserved. This program and the accompanying materials
+	are made available under the terms of the Eclipse Public License v1.0
+	which accompanies this distribution, and is available at
+	http://www.eclipse.org/legal/epl-v10.html
+
+-->
 <scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="binding.solarlog">
```

Comment:
This whole file should be deleted from the repository because the SolarLogHandlerFactory is using DS annotations.